apps/schematic/api/Dockerfile

```diff
@@ -1,41 +1,70 @@
-FROM python:3.10.12-slim-buster
+FROM tiangolo/uwsgi-nginx-flask:python3.10
 
-ENV APP_DIR=/opt/app
+# add version tag as a build argument
+ARG TAG
```

Comment:
How do tag behave when no default value is specified?

Reply:
Thanks for the feedbacks Thomas. I decided to remove this line since it is not really useful